TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -331,10 +319,38 @@ import PromiseKit
         linkedBrokersToRemove.forEach { linkedBrokerToRemove in
             self.removeBroker(linkedBroker: linkedBrokerToRemove)
         }
-        
+
+        // Sync accounts
+        self.linkedBrokers.forEach { localLinkedBroker in
+            remoteLinkedBrokers.first { remoteLinkedBroker in
+                localLinkedBroker.linkedLogin.userId == remoteLinkedBroker.userId
+            }.flatMap { remoteLinkedBroker in
+                syncAccounts(localLinkedBroker: localLinkedBroker, remoteLinkedBroker: remoteLinkedBroker)
```

Comment:
@guillaumedebavelaere are you sure? That means we'll need to read from the keychain every time. Yahoo plans to run this method every 30 seconds so it'll be very heavy.